itou/utils/command.py

```diff
@@ -48,15 +47,14 @@ def _command_duration_logger(command):
 
 
 @contextlib.contextmanager
-def _command_info_manager(command):
+def _command_info_manager(command, *, wet_run=None):
     command_info_before = get_current_command_info()
-    if command_info_before is None:
-        local.command_info = CommandInfo(str(command.run_uid), command.__class__.__module__)
+    run_uid = command_info_before.run_uid if command_info_before else uuid.uuid4()
+    local.command_info = CommandInfo(str(run_uid), command.__class__.__module__, wet_run)
```

Comment:
nit:
```suggestion
    run_uid = command_info_before.run_uid if command_info_before else str(uuid.uuid4())
    local.command_info = CommandInfo(run_uid, command.__class__.__module__, wet_run)
```
pour que `run_uid` soit toujours un `str`.

Et tu peux aussi supprimer `self.run_uid = uuid.uuid4()` du `__init__`